src/pages/data-management/index.jsx

```diff
@@ -54,6 +55,13 @@ const DataManagementPage = ({ route }) => {
   }, []);
 
   useEffect(() => {
+    if (activeProject?.experiments.length > 0) {
```

Comment:
what happens if experimens is undefined? can it happen?

Reply:
No, `experiments` default to an empty array